packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -603,6 +604,12 @@ class TextureLayer extends Layer {
   /// un-freezes it when it is certain that a frame with the new size is ready.
   final bool freeze;
 
+  /// Used to set the filterQuality of the texture
```

Comment:
please use full sentences in doc comments. They should end all with a `.`.

Also: The first sentence of a doc should be its own paragraph summarizing what's documented.